usage with apollo-server-lambda

When i try to use this package with `apollo-server-lambda` i get an error:
`ModuleNotFoundError: Module not found: Error: Can't resolve 'apollo-server' in '.../node_modules/@profusion/apollo-validation-directives/build/lib'`
no idea what is the difference in API between them and if it's even possible to use it with `apollo-server-lambda` (I havesn't seen any working examples so far), but it would be really nice thing

--- a/.eslintrc.json
+++ b/.eslintrc.json
@@ -30,6 +30,7 @@
         }
     },
     "rules": {
+        "import/no-extraneous-dependencies": ["error", {"devDependencies": ["examples/*.*"]}],
         "import/extensions": ["error", "never"],
         "@typescript-eslint/consistent-type-assertions": [
             "error",

--- a/examples/access-control-directives.ts
+++ b/examples/access-control-directives.ts
@@ -1,4 +1,6 @@
-import { gql, makeExecutableSchema, ApolloServer } from 'apollo-server';
+import { ApolloServer } from 'apollo-server';
+import { makeExecutableSchema } from 'graphql-tools';
+import gql from 'graphql-tag';
 
 import { auth, hasPermissions, MissingPermissionsResolverInfo } from '../lib';
 

--- a/examples/value-validation-directives.ts
+++ b/examples/value-validation-directives.ts
@@ -1,4 +1,6 @@
-import { makeExecutableSchema, ApolloServer, gql } from 'apollo-server';
+import { ApolloServer } from 'apollo-server';
+import { makeExecutableSchema } from 'graphql-tools';
+import gql from 'graphql-tag';
 import { ValidationError } from 'apollo-server-errors';
 
 import { graphql, print, GraphQLResolveInfo } from 'graphql';

--- a/lib/EasyDirectiveVisitor.test.ts
+++ b/lib/EasyDirectiveVisitor.test.ts
@@ -10,7 +10,8 @@ import {
   GraphQLBoolean,
 } from 'graphql';
 import { print } from 'graphql/language/printer';
-import { gql, makeExecutableSchema } from 'apollo-server';
+import { makeExecutableSchema } from 'graphql-tools';
+import gql from 'graphql-tag';
 
 import EasyDirectiveVisitor from './EasyDirectiveVisitor';
 

--- a/lib/EasyDirectiveVisitor.ts
+++ b/lib/EasyDirectiveVisitor.ts
@@ -15,7 +15,8 @@ import {
   print,
   printType,
 } from 'graphql';
-import { gql, SchemaDirectiveVisitor } from 'apollo-server';
+import gql from 'graphql-tag';
+import { SchemaDirectiveVisitor } from 'graphql-tools';
 
 export type ReadonlyGraphQLDirectiveConfigWithoutName = Readonly<
   {

--- a/lib/ValidateDirectiveVisitor.test.ts
+++ b/lib/ValidateDirectiveVisitor.test.ts
@@ -12,7 +12,8 @@ import {
   GraphQLResolveInfo,
 } from 'graphql';
 import { print } from 'graphql/language/printer';
-import { gql, makeExecutableSchema } from 'apollo-server';
+import gql from 'graphql-tag';
+import { makeExecutableSchema } from 'graphql-tools';
 import { ValidationError } from 'apollo-server-errors';
 
 import ValidateDirectiveVisitor, {

--- a/lib/auth.test.ts
+++ b/lib/auth.test.ts
@@ -1,6 +1,7 @@
 import { GraphQLResolveInfo, graphql } from 'graphql';
 import { print } from 'graphql/language/printer';
-import { gql, makeExecutableSchema } from 'apollo-server';
+import { makeExecutableSchema } from 'graphql-tools';
+import gql from 'graphql-tag';
 import { AuthenticationError } from 'apollo-server-errors';
 
 import AuthDirectiveVisitor from './auth';

--- a/lib/createValidateDirectiveVisitor.test.ts
+++ b/lib/createValidateDirectiveVisitor.test.ts
@@ -1,5 +1,6 @@
 import { graphql, GraphQLBoolean, GraphQLObjectType } from 'graphql';
-import { gql, makeExecutableSchema } from 'apollo-server';
+import { makeExecutableSchema } from 'graphql-tools';
+import gql from 'graphql-tag';
 
 import createValidateDirectiveVisitor from './createValidateDirectiveVisitor';
 import ValidateDirectiveVisitor from './ValidateDirectiveVisitor';

--- a/lib/foreignNodeId.test.ts
+++ b/lib/foreignNodeId.test.ts
@@ -1,6 +1,7 @@
 import { graphql } from 'graphql';
 import { print } from 'graphql/language/printer';
-import { gql, makeExecutableSchema } from 'apollo-server';
+import { makeExecutableSchema } from 'graphql-tools';
+import gql from 'graphql-tag';
 import { ValidationError } from 'apollo-server-errors';
 
 import { ValidateFunction } from './ValidateDirectiveVisitor';

--- a/lib/hasPermissions.test.ts
+++ b/lib/hasPermissions.test.ts
@@ -7,7 +7,8 @@ import {
   GraphQLResolveInfo,
 } from 'graphql';
 import { print } from 'graphql/language/printer';
-import { gql, makeExecutableSchema } from 'apollo-server';
+import { makeExecutableSchema } from 'graphql-tools';
+import gql from 'graphql-tag';
 import { ForbiddenError } from 'apollo-server-errors';
 
 import {

--- a/lib/listLength.test.ts
+++ b/lib/listLength.test.ts
@@ -1,5 +1,6 @@
 import { GraphQLSchema } from 'graphql';
-import { gql, makeExecutableSchema } from 'apollo-server';
+import gql from 'graphql-tag';
+import { makeExecutableSchema } from 'graphql-tools';
 import { ValidationError } from 'apollo-server-errors';
 
 import listLength from './listLength';

--- a/lib/pattern.test.ts
+++ b/lib/pattern.test.ts
@@ -1,5 +1,6 @@
 import { GraphQLSchema } from 'graphql';
-import { gql, makeExecutableSchema } from 'apollo-server';
+import gql from 'graphql-tag';
+import { makeExecutableSchema } from 'graphql-tools';
 import { ValidationError } from 'apollo-server-errors';
 
 import pattern from './pattern';

--- a/lib/range.test.ts
+++ b/lib/range.test.ts
@@ -1,5 +1,6 @@
 import { GraphQLSchema } from 'graphql';
-import { gql, makeExecutableSchema } from 'apollo-server';
+import gql from 'graphql-tag';
+import { makeExecutableSchema } from 'graphql-tools';
 import { ValidationError } from 'apollo-server-errors';
 
 import range from './range';

--- a/lib/selfNodeId.test.ts
+++ b/lib/selfNodeId.test.ts
@@ -1,6 +1,7 @@
 import { graphql } from 'graphql';
 import { print } from 'graphql/language/printer';
-import { gql, makeExecutableSchema } from 'apollo-server';
+import gql from 'graphql-tag';
+import { makeExecutableSchema } from 'graphql-tools';
 import { ValidationError } from 'apollo-server-errors';
 
 import SelfNodeId from './selfNodeId';

--- a/lib/stringLength.test.ts
+++ b/lib/stringLength.test.ts
@@ -1,5 +1,6 @@
 import { GraphQLSchema } from 'graphql';
-import { gql, makeExecutableSchema } from 'apollo-server';
+import gql from 'graphql-tag';
+import { makeExecutableSchema } from 'graphql-tools';
 import { ValidationError } from 'apollo-server-errors';
 
 import stringLength from './stringLength';

--- a/lib/test-utils.test.ts
+++ b/lib/test-utils.test.ts
@@ -4,7 +4,7 @@ import {
   ExecutionResultDataDefault,
   ExecutionResult,
 } from 'graphql/execution/execute';
-import { gql } from 'apollo-server';
+import gql from 'graphql-tag';
 
 import EasyDirectiveVisitor from './EasyDirectiveVisitor';
 

--- a/package.json
+++ b/package.json
@@ -37,7 +37,7 @@
     "@types/jest": "^26.0.7",
     "@typescript-eslint/eslint-plugin": "^3.7.1",
     "@typescript-eslint/parser": "^3.7.1",
-    "apollo-server": "^2.16.1",
+    "apollo-server": "^2.17.0",
     "eslint": "^7.5.0",
     "eslint-config-airbnb-base": "^14.2.0",
     "eslint-config-prettier": "^6.11.0",
@@ -56,8 +56,11 @@
     "typescript": "^3.9.7"
   },
   "peerDependencies": {
-    "apollo-server": "^2.16.1",
     "apollo-server-errors": "^2.4.2",
     "graphql": "^14.7.0"
+  },
+  "dependencies": {
+    "graphql-tag": "^2.11.0",
+    "graphql-tools": "^4.0.0"
   }
 }

